src/d3a/d3a_core/area_serializer.py

```diff
@@ -163,7 +163,6 @@ def area_from_string(string, config=None):
 
 
 def are_all_areas_unique(area, list_of_areas=set()):
```

Comment:
I think it is much better to remove the call of this function rather than remove the assertion from inside it. That way the recursion that this function includes will be avoided.

Reply:
Yes Spyros ,I have removed completely from both files, I thought function call will be utilised in other way to us in future to add areas like dummy in backend,